### DISCUSSION TITLE
Transfer asdf-coder

### DISCUSSION THREAD

**Discussion by toshikish** [mise-plugins/.github#8] [Transfer Request]
I made a plugin for [Coder](https://coder.com/) CLI, a self-hosted remote development environment.
It is hosted under [toshikish/asdf-coder](https://github.com/toshikish/asdf-coder).

I would like to transfer it to mise-plugins.

**Comment by jdx** [comment]
go ahead and send it to me and I'll move it to the org

**Reply by toshikish** [reply]
Thanks.
I just transferred it to you.
